src/modules/translation/languages/en.json

```diff
@@ -953,6 +955,12 @@
   },
   "third_party_collection_detail_page": {
     "notice": "Each published item takes 1 Slot. {buy_link} to buy Slot Packs and start publishing your creations!",
+    "search_placeholder": "Search {count} {count, plural, one {item} other {items}}...",
+    "search_info": "{page}-{pageTotal} of {total}",
+    "clear_selection": "Clear selection",
+    "selection": "{count, plural, one {# item} other {# items}} selected.",
+    "max_select_count": "You can publish up to {count} items at a time",
+    "click_here": "Click here",
```

Comment:
This seems to not be being used (the global one does though).

Reply:
Fricking rebase